web/.env.example

```
@@ -1,4 +1,5 @@
 RESEND_API_KEY=
+CMUX_BILLING_RECOVERY_RATE_LIMIT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CMUX_BILLING_RECOVERY_RATE_LIMIT_ID key should go before the RESEND_API_KEY key

(UnorderedKey)